leaflet/index.d.ts

```diff
@@ -1339,11 +1394,14 @@ declare namespace L {
         scrollWheelZoom: Handler;
         tap: Handler;
         touchZoom: Handler;
-    }
 
-    export function map(id: string, options?: MapOptions): Map;
+        options: MapOptions;
+    }
 
-    export function map(el: HTMLElement, options?: MapOptions): Map;
+    /**
+     * ID of a HTML-Element as string or the HTML-ELement itself
+     */
+    export function map(element: string | HTMLElement, options?: MapOptions): Map;
```

Comment:
The same as above, string would not be the content, it is the ID of the element...